packages/neuron-ui/src/components/SpecialAssetList/index.tsx

```diff
@@ -416,6 +428,19 @@ const SpecialAssetList = () => {
                   )
                 }
 
+                if (isSpore) {
+                  return (
+                    <div className={styles.actionBtnBox}>
+                      <Button
+                        type="cancel"
```

Comment:
It may be a historical problem that `cancel` is not suitable for a `view` action, probably `default` or `normal`, etc.

It can be optimized later.

Reply:
I've removed the "cancel" type